org.recoll.recoll.yml

```diff
@@ -0,0 +1,125 @@
+id: org.recoll.recoll
+runtime: org.kde.Platform
+runtime-version: 5.15-23.08
+sdk: org.kde.Sdk
+base: io.qt.qtwebkit.BaseApp
+base-version: 5.15-23.08
+command: recoll
+rename-desktop-file: recoll-searchgui.desktop
+rename-icon: recoll
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=home
+  - --env=ASPELL_CONF=dict-dir /app/share/dicts; local-data-dir /app/share/datadicts
+
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+
+modules:
+  - name: xapian-core
+    config-opts:
+      - --disable-sse
+      - --disable-backend-chert
+      - --disable-backend-remote
+      - --disable-documentation
+    cleanup:
+      - /bin
+      - /share/aclocal
+      - /share/doc
+      - /share/man
```

Comment:
these cleanups should be toplevel so that you don't have to repeat for everything. same for /include, `*.a, *.la` etc.